incubator/cassandra/values.yaml

```diff
@@ -0,0 +1,44 @@
+## Cassandra image version
+## ref: https://hub.docker.com/r/library/cassandra/
+image:
+  repo: "cassandra"
+  tag: "latest"
+  pullPolicy: IfNotPresent
+
+## Specify a service type
+## ref: http://kubernetes.io/docs/user-guide/services/
+service:
+  type: ClusterIP
+
+## Persist data to a persitent volume
+persistence:
+  enabled: true
+  storageClass: generic
+  accessMode: ReadWriteOnce
+  size: 100Gi
+
+## Configure resource requests and limits
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+resources:
+  requests:
+    memory: 10Gi
```

Comment:
Ok, I reduce a size of default memory to 4Gi and 2 vCPU as it is very minimum specification for development. Also add more node in `values.yaml` for recommending settings on production. :)